app/assets/stylesheets/components/_slider.scss

```diff
@@ -0,0 +1,88 @@
+$track-w: 16em;
+$track-h: .625em;
+$track-bg: $grey-4;
+$track-fill: linear-gradient(90deg, $grey-2 25%, $grey-2 100%);
+
+$thumb-d: 1.5em;
```

Comment:
It means diameter. Agree, but will wait until design have taken a look before finalising the CSS.